Library/Formula/crawl.rb

```diff
@@ -0,0 +1,37 @@
+class Crawl < Formula
+  desc "Dungeon Crawl: Stone Soup"
+  homepage "http://crawl.develz.org/"
+  version "0.17.1"
+
+  url "https://github.com/crawl/crawl.git",
+    :shallow => false,
+    :revision => "942a3d2973debefdaddbf6291e4ebc890725b920"
```

Comment:
Yes, it uses submodules—the tarball from GitHub doesn't include them.

I originally tagged the version, but it complained; I then tried removing version from the formula, but because it's below for checking the installation output, I was getting errors on that. I'll try poking at that again.